io/plugins/eu.esdihumboldt.hale.io.jdbc/schema/eu.esdihumboldt.hale.io.jdbc.config.exsd

```diff
@@ -176,6 +176,30 @@
                </appinfo>
             </annotation>
          </attribute>
+         <attribute name="isSchemaSelectionEnable" type="boolean" use="required">
```

Comment:
One can add only one choice in any Element and we already added one Choice for prefix and which is unbounded, So I thought I should not add in to that Choice otherwise Schema-selection also be unbounded.